docs/en-us/dev/user_doc/guide/alert/enterprise-wechat.md

```diff
@@ -1,29 +1,9 @@
 # Enterprise WeChat
 
-If you need to use Enterprise WeChat to alert, please modify the `alert.properties` file after the installation is complete, and then restart the alert service. The configuration example of enterprise WeChat is as follows
+If you need to use Enterprise WeChat to alert, please create alarm Instance in warning instance manage, and then chose the wechat plugin. The configuration example of enterprise WeChat is as follows
 
-```shell
-# Enable enterprise WeChat alarm or not: ``true`` or ``false``
-enterprise.wechat.enable="true"
+<img src="enterprise-wechat-plugin.png">
 
-# Enterprise WeChat corpid, each enterprise has a unique corpid, to obtain this information, you can view the "enterprise ID" under "My Company"-"Enterprise Information" in the management background (administrator rights are required)
-enterprise.wechat.corp.id="xxx"
-
-# Secret for Enterprise WeChat, secret is the "key" used to ensure data security in enterprise applications, and each application has an independent access key
-enterprise.wechat.secret="xxx"
-
-# Enterprise WeChat agentid, each application has a unique agentid. In the management background -> "Apps and applets" -> "Apps", click on an application, you can see the agentid
-enterprise.wechat.agent.id="xxxx"
-
-# Userid when notification send to Enterprise WeChat, multiple separated by commas. Each member has a unique userid, the so-called "account". In the management background -> "Contacts" -> click on a member's details page, you can see
-enterprise.wechat.users=zhangsan,lisi
-
-# Get the address of access_token, modify if you need
-enterprise.wechat.token.url=https://qyapi.weixin.qq.com/cgi-bin/gettoken?corpid={corpId}&corpsecret={secret}
-
-# Send application message address, modify if you need
-enterprise.wechat.push.url=https://qyapi.weixin.qq.com/cgi-bin/message/send?access_token={token}
-
-# Message format
-enterprise.wechat.user.send.msg={\"touser\":\"{toUser}\",\"agentid\":\"{agentId}\",\"msgtype\":\"markdown\",\"markdown\":{\"content\":\"{msg}\"}}
-```
+Where send Type corresponds to app and appchat respectively:
+APP: https://work.weixin.qq.com/api/doc/90000/90135/90236
```

Comment:
Is there a documnet in English? I think English is better but I could not find it?

Reply:
https://developers.weixin.qq.com/community/develop/doc/000ea2cdb84d9009c9ea7c7b651400
There is no English version of enterprise wechat API documents.

Reply:
Ok, thanks for the clarification 👍 . But I still want to 👎  for wechat enterprise API doc